Slambook/index.js

```diff
@@ -39,7 +39,7 @@ let nicknameValue= document.querySelector(`[name='nickname']`).value;
 let songForMeValue= document.querySelector(`[name='songForMe']`).value; 

```

Comment:
The variable name 'formValues' should follow camel case convention and be written as 'formValues'